applications/fishing-map/public/locales/source/translations.json

```diff
@@ -238,43 +221,6 @@
     "save": "Save the current view",
     "saved": "Current view was saved and it's available in your user profile",
     "saveLogin": "You need to login to save views",
-    "title_plural": "Workspaces",
-    "title": "Workspace"
-  },
-  "timebar": {
-    "playback": {
-      "playAnimation": "Play animation",
-      "pauseAnimation": "Pause animation",
-      "toogleAnimationLooping": "Toggle animation looping",
-      "moveBack": "Move back",
-      "moveForward": "Move forward",
-      "changeAnimationSpeed": "Change animation speed"
-    },
-    "timerange": {
-      "title": "Select a time range",
-      "start": "start",
-      "end": "end",
-      "last30days": "Last 30 days",
-      "done": "Done",
-      "errorEarlyStart": "Your start date is the earliest date with data available",
-      "errorLatestEnd": "Your end date is the latest date with data available",
-      "errorMinRange": "Your start and end date must be at least one day apart",
-      "errorMaxRange": "Your time range is the maximum range with data available"
-    },
-    "bookmark": {
-      "goToBookmark": "Go to your bookmarked time range",
-      "deleteBookmark": "Delete time range bookmark"
-    },
-    "dragLabel": "Drag to change the time range",
-    "lastUpdate": "Last update",
-    "setBookmark": "Bookmark current time range",
-    "day": "day",
-    "year": "year",
-    "month": "month",
-    "hour": "hour",
-    "zoomIn": "Zoom in",
-    "zoomTo": "Zoom to",
-    "zoomOut": "Zoom out",
-    "selectTimeRange": "Select a time range"
```

Comment:
I suspect all of these are still in use. What could we do so that i18next-parser does not remove them?

Reply:
yes I see lot of keys that doend't feel that were removed, did you try some scanner plugins? like
https://github.com/i18next/i18next-scanner/
there are more @rdgfuentes

Reply:
As the parser only reads `src` code to find keys... there are some use cases where the key isn't in the code but needs to appear in sources, for example:
- API dynamic fields from vessels ([workaround](https://github.com/GlobalFishingWatch/frontend/blob/e7809a4a7977ef62b554f1f29a3a85cad5a86509/applications/fishing-map/src/features/workspace/vessels/VesselTranslations.ts))
- Timebar translations, we don't use the keys in our code but pass the object configuration into the component. How could we manage this ?

Another idea is to update the crowdin sync to update the sources and allow deleting from there too.

Anyway I'd do a round of cleaning in this PR to check which one are false positives and try to keep the source as clean as possible

Reply:
@fertandil87 I tried it briefly but could not make it work with Typescript. Maybe this other one https://www.npmjs.com/package/i18next-scanner-typescript could work but have not tried it.

@j8seangel It's something to think about...  Thanks